### PR TITLE
TGUI PTL fixes

### PR DESCRIPTION
**Pull request by Sovexe**
## About the PR 
- Fixes PTL laser lacking power updates
- Fixes emagging PTL
- Emagged PTL output dial becomes bipolar and has a red line instead of blue when at -1 or lower

## Why's this needed? 
bugs

### CODE REVIEW

File: code/modules/power/pt_laser.dm [hunk 1]
```diff
@@ -332,12 +332,11 @@
 		terminal.powernet.newload += amount
 
 /obj/machinery/power/pt_laser/proc/update_laser_power()
-	if(abs(output) > charge)
+	if(abs(src.output) > src.charge && src.firing)
```
Comment (ZeWaka) [inline code/modules/power/pt_laser.dm:335]: ambiguous

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/modules/power/pt_laser.dm (hunk 1 above)
Comment (Sovexe) [reply code/modules/power/pt_laser.dm:335]: `update_laser_power()` is called anytime `src.output` is changed. If the laser has already ceased firing there is no reason to call `stop_firing()` again for each adjustment of output that is still greater than the charge.

Comment (ZeWaka) [reply code/modules/power/pt_laser.dm]: I meant with parenthesis lol

Comment (Sovexe) [reply code/modules/power/pt_laser.dm]: oh I added those too

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**